packs/redis/config.yml

```diff
@@ -7,9 +7,26 @@ name: redis
 title: Redis
 
 # Description of the pack
-description: |
-  Official New Relic pack for Redis
-  Use this pack together with the New Relic Redis On Host Integration to get insight into the performance of your Redis instances.
+description: |+
+  ## A Complete Redis Monitoring System
```

Comment:
2 h2's, maybe result of merge conflict? gotta love those merges

Reply:
Solved in new commit